pkgs/development/python-modules/pytest-flask/default.nix

```diff
@@ -12,7 +13,7 @@ buildPythonPackage rec {
   doCheck = false;
 
   buildInputs = [
```

Comment:
since this won't work without pytest also being available
```suggestion
  propagatedBuildInputs = [
```

Reply:
done